network/manager.go

```diff
@@ -425,6 +425,7 @@ func (nm *networkManager) CreateEndpoint(cli apipaClient, networkID string, epIn
 func (nm *networkManager) UpdateEndpointState(ep *endpoint) error {
 	ifnameToIPInfoMap := generateCNSIPInfoMap(ep) // key : interface name, value : IPInfo
 	logger.Info("Calling cns updateEndpoint API with ", zap.String("containerID: ", ep.ContainerID), zap.String("HnsId: ", ep.HnsId), zap.String("HostIfName: ", ep.HostIfName))
+	logger.Info("ifnameToIPInfoMap:", zap.Any("ifnameToIPInfoMap", ifnameToIPInfoMap))
```

Comment:
do we need to log out the entire map? also usage of `zap.Any` is not reccomended 🥲

Reply:
I was using it for debugging purposes on my tests. It's removed